packages/mui-docs/src/NProgressBar/NProgressBar.js

```diff
@@ -94,7 +38,52 @@ function NProgressBar(props) {
   return (
     <NoSsr>
       {props.children}
-      <GlobalStyles />
+      <GlobalStyles
+        styles={(theme) => ({
+          '#nprogress': {
+            direction: 'ltr',
+            pointerEvents: 'none',
+            position: 'fixed',
+            top: 0,
+            left: 0,
+            right: 0,
+            height: 2,
+            zIndex: (theme.vars || theme).zIndex.tooltip,
```

Comment:
Could this be `(theme.vars || theme).zIndex?.tooltip` ? `theme.vars.zIndex` is undefined
![Screenshot 2023-02-17 at 9 05 02 PM](https://user-images.githubusercontent.com/4997971/219661199-ab693f82-e815-4d8e-aa07-9140613fd33c.png)

Reply:
Ahhh, you are right. On the other hand, I think we could add `zIndex` to Joy UI as well.

Reply:
It seems to be a theming isolation issue rather than a bug with NProgress.